package.json

```diff
@@ -22,15 +23,17 @@
     "react-native-bottom-sheet": "1.0.1",
     "react-native-button": "1.8.2",
     "react-native-cookies": "3.1.0",
-    "react-native-device-info": "0.10.2",
+    "react-native-device-info": "0.11.0",
     "react-native-drawer": "2.3.0",
     "react-native-exception-handler": "1.1.0",
     "react-native-image-picker": "jp928/react-native-image-picker#6ee35b69f3dbd6c7c66f580fd4d9eabf398703d4",
     "react-native-keyboard-aware-scroll-view": "0.2.8",
     "react-native-linear-gradient": "2.0.0",
+    "react-native-local-auth": "enahum/react-native-local-auth.git",
```

Comment:
One of these days we really need to move more of these forks out of our own accounts and into the mattermost org :p